src/services/findUserById.ts

```diff
@@ -0,0 +1,6 @@
+import usersFromServer from '../api/users';
+import { User } from '../types/user';
+
+export const findUserById = (id: number): User => {
+  return usersFromServer.find(user => user.id === id) as User;
```

Comment:
better to avoid the assertions in TS
```suggestion
  return usersFromServer.find(user => user.id === id) || null;
```